tests/test_chapter_processing.py

```diff
@@ -235,37 +234,19 @@ def test_process_chapter_guessing_datatypes(tmp_path, datatype):
             ("author_bio", "afterword")
             ]
         )
-def test_process_chapter_guessing_datatypes_non_allowed(tmp_path, datatype):
+def test_process_chapter_guessing_datatypes_inferred(tmp_path, datatype):
     """
     confirm that the default front/backmatter data types are applied
-    in the case that we get a guessed-at front/backmatter chapter title
+    in the case that we get an inferred front/backmatter chapter title
     """
     test_env = tmp_path / 'tmp'
-    test_out = test_env / 'output'
+    test_out = test_env / 'build'
     test_env.mkdir()
     test_out.mkdir()
     test_file_path = test_env / f'{datatype[0]}.html'
-    shutil.copytree('tests/example_html', test_env, dirs_exist_ok=True)
-    shutil.move(test_env / 'intro.html', test_file_path)
+    shutil.copy('tests/example_book/_build/html/intro.html', test_file_path)
     process_chapter(test_file_path, test_out)
     # the resulting section should have a data-type of "datatype"
     with open(test_out / f'{datatype[0]}.html') as f:
         text = f.read()
         assert text.find(f'data-type="{datatype[1]}') > -1
-
-
-def test_chapter_process_confirm_remove_span(tmp_path):
```

Comment:
I removed this test since it was a duplicate of one of the above, and I don't remember what it was trying to test for.